index.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Gardener</title>
+  <link rel="stylesheet" href="css\style.css">
+</head>
+<body>
+  <header id="header">
+    <div class="container flex-box">
+      <h1 class="logo"><a href=""><img src="images\logo.png" alt="Gardener"></a></h1>
+      <nav>
+        <ul class="flex-box">
+              <li><a href="#home">Home</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="nav-icon">
+        <div></div>
+      </div>  
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="banner">
+      <div class="container flex-box">
+        <div class="panel-left">
```

Comment:
Should apply full width in mobile view. Refer design.

Reply:
Still, this issue is there due to margin property from our recent projects area.